app/helpers/ama_layout_navigation_helper.rb

```diff
@@ -0,0 +1,28 @@
+module AmaLayoutNavigationHelper
```

Comment:
The path of the file would change to `app/helpers/ama_layout/navigation_helper.rb` as well.